doc/command_ref.rst

```diff
@@ -142,8 +142,30 @@ Options
 ``-h, --help``
     Show the help.
 
+.. _installroot-label:
+
 ``--installroot=<path>``
-    set install root
+    Specifies an alternative installroot, relative to where all packages will be
+    installed. Think of this like doing ``chroot <root> dnf`` except using
+    ``--installroot`` allows dnf to work before the chroot is created. The
+    following things are taken from or stored in installroot: cachedir, gpgkey
+    (they are imported into installroot from path described in .repo file that
+    is not relative to installroot), config file (unless
+    ``--config=<config file>`` command-line option is used, then the path is not
+    relative to installroot), log files, releasever, repository, pluginconfpath.
+    The pluginpath is not relative to installroot. Note: You may also want to
+    use the command-line option ``--releasever=<release>`` when creating the
+    installroot otherwise the $releasever value is taken from the rpmdb within
+    the installroot (and thus it is empty at time of creation, the transaction
+    will fail).
+    If installroot is run for first time, repository, releasever, and dnf.conf
+    file is not present in installroot. Therefore the command-line option
+    ``--repofrompath <repo>,<path/url>`` can be used to define repository or
+    repository can be imported into installroot like with command
+    `dnf config-manager`_ . Then command like
+    ``dnf install --installroot --releasever=<release> package`` will install
+    package and setup releasever in installroot. Subsequent commands will take
+    releasever from installroot.
```

Comment:
@jsilhan Is that behavior like it is expected, or shall I refactor it or add some tests?